crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -1363,14 +1365,22 @@ impl ScheduleGraph {
                 let access_a = system_a.component_access();
                 let access_b = system_b.component_access();
                 if !access_a.is_compatible(access_b) {
-                    let conflicts: Vec<_> = access_a
-                        .get_conflicts(access_b)
-                        .into_iter()
-                        .filter(|id| !ignored_ambiguities.contains(id))
-                        .collect();
-
-                    if !conflicts.is_empty() {
-                        conflicting_systems.push((a, b, conflicts));
+                    match access_a.get_conflicts(access_b) {
+                        AccessConflict::Individual(conflicts) => {
+                            let conflicts: Vec<_> = conflicts
+                                .ones()
+                                .map(|i| ComponentId::get_sparse_set_index(i))
+                                .filter(|id| !ignored_ambiguities.contains(id))
+                                .collect();
+                            if !conflicts.is_empty() {
+                                conflicting_systems.push((a, b, conflicts));
+                            }
+                        }
+                        AccessConflict::All => {
+                            // there is no specific component conflicting, but the systems are overall incompatible
+                            // for example 2 systems with `Query<EntityMut>`
+                            conflicting_systems.push((a, b, Vec::new()));
```

Comment:
this is not ideal.. maybe we'd like to introduce a similar enum in the schedule graph conflicts? not sure if that's what we want

Reply:
IMO this can wait until follow-up.

Reply:
Out of curiousity, what happens with the reporting in this case?

Reply:
It says that the two systems are ambiguous and conflict on `World`: https://github.com/cBournhonesque/bevy/blob/944cc182d2265df2165ac03372d681d718e1723d/crates/bevy_ecs/src/schedule/schedule.rs#L1883-L1883